### DISCUSSION TITLE
Branching

### DISCUSSION THREAD

**Discussion by DeeJayTC** [thomasduft/microwf#23] [Ideas]
Hey there,
first of all thanks for your great work so far. Really helped me get a few things going a lot faster than expected :)

I'm wondering if there are any plans (or if its already working and im just too stupid to find out) for workflows with multiple branches.

Especially interested in how we could build workflows where multiple steps have to be approved by more than one person. 

Tim

**Comment by thomasduft** [comment]
Hi @DeeJayTC

First of all thanks and your compliments are very much appreciated.

Your question might fall in the category of this closed issue #20. If your use case really requires branching to sub-workflows you will need to write some transition logic within the `CanMakeTransition / BeforeTransition`-hook in your main workflow that checks whether all of your sub-workflows have been finished.

If possible you could also provide some basic instructions that showcases your use case in a similar manner the mentioned issues did. 

Have a nice day.

Thomas
